Makefile

```diff
@@ -257,13 +257,26 @@ test-integration: .init build
 	contrib/hack/setup-kubectl.sh
 	contrib/hack/test-apiserver.sh
 
-clean:
+clean: clean-bin clean-deps clean-build-image clean-generated clean-coverage
+
+clean-bin:
 	rm -rf $(BINDIR)
-	rm -f .init .scBuildImage .generate_files .generate_exes
-	rm -f $(COVERAGE)
-	find $(TOP_SRC_DIRS) -name zz_generated* -exec rm {} \;
+	rm -f .generate_exes
+
+clean-deps:
+	rm -f .init
+
+clean-build-image:
+	rm -f .scBuildImage
 	docker rmi -f scbuildimage > /dev/null 2>&1 || true
 
+clean-generated:
```

Comment:
But that wasn't getting cleaned before, right? I don't see where it was. Am I overlooking it? Not saying we shouldn't... but maybe it's a separate PR?